sbin/devd/devmatch-openrc.conf

```diff
@@ -9,7 +9,7 @@
 #
 # Generic NOMATCH event
 nomatch 100 {
-	action "sh /etc/rc.devd devmatch start.'?'$_ '?'$_";
+       action "/etc/rc.d/devmatch quietstart '?'$_";
```

Comment:
This would break our implementation, since rc.devd handles the openrc specific devmatch stuff.

Reply:
Removing this change should let us merge the patch to fix normal rc.d usage.

Reply:
So when debugging I found one wrinkle: devmatch.conf is not getting triggered but devmatch-openrc.conf is and when  it is triggered the action fails saying "exec: /devmatch not found". This seems like /sbin/devmatch is not in the path somehow when launched via the above subshell... 

> 
> Executing 'sh /etc/rc.devd devmatch start.'?'$' at bus=0 hubaddr=3 port=4 devaddr=9 interface=0 ugen=ugen0.9 vendor=0x0451 product=0xca01 devclass=0x00 devsubclass=0x00 devproto=0x00 sernum="" release=0x0100 mode=host intclass=0x03 intsubclass=0x00 intprotocol=0x00 on uhub2' '?'$' at bus=0 hubaddr=3 port=4 devaddr=9 interface=0 ugen=ugen0.9 vendor=0x0451 product=0xca01 devclass=0x00 devsubclass=0x00 devproto=0x00 sernum="" release=0x0100 mode=host intclass=0x03 intsubclass=0x00 intprotocol=0x00 on uhub2''
> exec: /devmatch: not found

Also only I was only able to get action triggered running devd -d, in foreground, no events are getting here via rc service... (currently under investigation)

Reply:
Fixed in the following commit https://github.com/trueos/trueos/pull/284/commits/bfce73f8e3a5560023acf60dc03441239b8d6d7c